app/presenters/track.rb

```diff
@@ -47,17 +47,17 @@ def topic_content(topic_key)
       end
 
       def contributing_instructions(topic_key)
-        content_with_trackler_data("./x/docs/md/track/BETTER.md").gsub('TOPIC', topic_key.to_s.upcase)
+        populate_template_with_trackler_data(File.read("./x/docs/md/track/BETTER.md")).gsub('TOPIC', topic_key.to_s.upcase)
```

Comment:
I'm mostly worried about the fallback file path and file name format being defined in two places.